app/templates/server/server.js

```diff
@@ -11,10 +11,12 @@ const appName = require('./../package').name;
 const http = require('http');
 const express = require('express');
 const log4js = require('log4js');
+const DEFAULT_LOG_LEVEL = log4js.levels.INFO;
```

Comment:
I prefer having a `process.env.LOG_LEVEL ||` at the front so it will take in an environment variable for user to override log level.
And, just combine the 2 lines you changed here.
`logger.level = process.env.LOG_LEVEL || log4js.levels.INFO;`